aten/src/ATen/TensorIterator.cpp

```diff
@@ -1388,7 +1388,7 @@ bool TensorIteratorBase::fast_set_up(const TensorIteratorConfig& config) {
     case FastSetupType::NON_OVERLAPPING_DENSE:
       {
         // find the index of a defined tensor in operands_ start from input tensor
-        int i_defined; // NOLINT(cppcoreguidelines-init-variables)
+        int i_defined = 0;
```

Comment:
Is this the best default initialization value? Maybe something more catastrophic like int min or something?

Reply:
Change to -1, which is better here.

Reply:
Change to -1, which is better here.